spring-petclinic-microservices/spring-petclinic-customers-service/pom.xml

```diff
@@ -38,12 +43,12 @@
         <dependency>
             <groupId>com.azure</groupId>
             <artifactId>azure-spring-data-cosmos</artifactId>
-            <version>3.2.0</version>
+            <version>3.16.0</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;dependency} -->
```

Comment:
same here, we should manage all dependency versions in the parent pom.

Reply:
Why are we still using spring-data-cosmos instead of our data starter?

Reply:
good question, I did't decide to do code change in this PR.

Reply:
Either way is okay with me, but do remember to change to the starter.